vsts/scripts/pullAndTag.sh

```diff
@@ -30,7 +30,7 @@ done <"$1"
 # Extract language string from string (e.g extract 'python' from 'category=python')
 if [ -n "$TESTINTEGRATIONCASEFILTER" ];then
 	# For DB tests we want all the runtime images to be present at thae agent machine
-	if [[ "$TESTINTEGRATIONCASEFILTER" != *db* ]];then
+	if [[ "$TESTINTEGRATIONCASEFILTER" != *db* && "$TESTINTEGRATIONCASEFILTER" != *buster*  ]];then
```

Comment:
need to update the comment: logic is to run all the integrations tests for buster  based build image against all the run time images, so we need to pull all the runtime images here just like we do for db tests